src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -266,8 +266,13 @@ private void setCcsTestMode(IInternalContest contest, boolean ccsTestMode) {
         contestInformation.setCcsTestMode(ccsTestMode);
         contest.updateContestInformation(contestInformation);
     }
-    
-    
+
+    private void LoadSampleJudgesData(IInternalContest contest, boolean loadSampleFiles) {
```

Comment:
Totally minor, picky point:  java method names should start with a lower-case letter.

Reply:
Not something that should hold up approval; just sayin'....